### PR TITLE
Add insight triage example

### PR DESCRIPTION
**Pull request by dbravovmw**

### CODE REVIEW

File: cli-plugins/insight/triaging-vulnerabilities.hbs.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+in the Supply Chain Security Tools - Store. Keep in mind this is an experimental feature, and the
+API is prone to changes in subsequent releases.
+
+## <a id='triage-description'></a>What is triaging?
+
+Vulnerability analysis, triaging or triage is the process of evaluating a reported vulnerability in
+order to determine an effective remediation plan. It is through triage that app teams are able to
+generate useful insights about the vulnerabilities in their software so that they can make the right
+decisions about when and how to mitigate them. The current implementation of triage follows
+[CycloneDX's Vulnerability Exploitability eXchange](https://cyclonedx.org/capabilities/vex/)
```
Comment (julia-pu) [inline cli-plugins/insight/triaging-vulnerabilities.hbs.md]: Exchange looks like the capitalization is on the wrong letter

File: cli-plugins/insight/triaging-vulnerabilities.hbs.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  --img-digest $TARGET_IMAGE
+```
+
+Where:
+- `TRIAGE_UID` is the uid of an existing analysis
+- `TARGET_IMAGE` is the digest of an image you want to copy the analysis to
+
+Keep in mind that there are a couple of conditions for this action to work:
+
+1. If specified, the targeted image or source must contain the package afected by the vulnerability
```
Comment (julia-pu) [inline cli-plugins/insight/triaging-vulnerabilities.hbs.md]: typo for `affected`

File: cli-plugins/insight/triaging-vulnerabilities.hbs.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+Where:
+
+- `CVEID` is the unique identifier of the vulnerability
+- `PKG_NAME` and `PKG_VERSION` are the name and version of the Application/OS package affected by the vulnerability
+- `IMG_DIGEST` is the digest of the image that contains the affected Application/OS package
+- `AG_UID` is the unique identifier for the workload that contains the image
+  - If your workload was deployed with Tanzu CLI, you can find its unique identifier with the command:
+    ```console
+    kubectl get workload $MY_WORKLOAD_NAME --namespace $MY_WORKLOAD_NAMESPACE --output jsonpath='{.metadata.uid}DDDDDDDDDDD'
```
Comment (Pivotal-Jeff-Jun) [inline cli-plugins/insight/triaging-vulnerabilities.hbs.md]: do we want to keep this special characters?

File: cli-plugins/insight/triaging-vulnerabilities.hbs.md (hunk 1 above)
Comment (dbravovmw) [reply cli-plugins/insight/triaging-vulnerabilities.hbs.md]: That is intended, since that's the definition of VEX. I'll add the acronym to make it clearer

**Review by julia-pu — APPROVED** [review]

File: cli-plugins/insight/triaging-vulnerabilities.hbs.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+results by image or source. For more information, use the `--help` flag or read through
+[this doc](./cli-docs/tanzu_insight_triage_list.md).
+
+## <a id='copying-analysis'></a>Copying an analysis
+
+Sometimes, you might run into scenarios where an existing analysis could be shared between multiple
+images, for example, when a new version of an existing image gets deployed by your workload and it
+contains the same vulnerability as the previous version, or when you create an analysis for an image
+that is shared between multiple workloads.
+
```
Comment (miketeve) [inline cli-plugins/insight/triaging-vulnerabilities.hbs.md:87]: Add the following if you think this is appropriate:`>**Note** The responsibility of assessing a vulnerability's impact is up to the person assessing the vulnerability.  Images and sources with the same package and version may be utilizing the package in a different manner and, thus, may not have the same analysis values.

Comment (dbravovmw) [reply cli-plugins/insight/triaging-vulnerabilities.hbs.md:87]: I've added this to the end of the section

**Pull request closed without merging**